File: coinflip.py
import random
head = 0
tail = 0
dif = 0
time = int(input("how many flips do you need?: "))
#flipping the coin
for i in range(time):
    ran = random.randint(0,1)
    if ran == 0:
        head += 1
    else:
        tail += 1
if head > tail:
    dif = head - tail
    cond = "head wins"
elif tail > head:
    dif = tail - head
    cond = "tail wins"
else:
    cond = "draw"
print("Number of heads:" + str(head))
print("Number of tails:" + str(tail))
print(cond)
print("difference: " + str(dif))

File: coinflip.c
#include <time.h>
#include <stdlib.h>
#include <stdio.h>

int main() {
    srand(time(NULL));// Initialization, should only be called once.
    int times;
    int head = 0;
    int tail = 0;
    int dif = 0;
    printf("how many times shoud I flip the coin: ");
    scanf("%d", &times);

    //flipping the coin
    for(int i = 0; i < times ; i++){
        int r = rand() % 2; // Returns a pseudo-random integer between 0 and 1.
        if(r == 0){
            head++;
        }
        else{
            tail++;
        }
    }

    printf("\nNumber of heads: %d", head);
    printf("\nNumber of tails: %d", tail);
        if(head > tail){
        printf("\nhead wins");
        dif = head - tail;
    }
    else if(tail > head){
        printf("\ntail wins");
        dif = tail - head;
    }
    else{
        printf("\ndraw");
    }
    printf("\nThe difference was %d", dif);
    return 0;
}